NEWS.md

```diff
@@ -5,6 +5,24 @@ changelog, see the [CHANGELOG] for each version via the version links.
 
 [CHANGELOG]: https://github.com/thoughtbot/scenic/commits/master
 
+## [1.1.2] - February 5, 2016
+
+### Fixed
+- Materialized view indexes are now properly dumped to `db/schema.rb`. This was
+  an oversight in previous releases, meaning `rake db:schema:load` was missing
+  indexes.
+- Calling `update_view` for a materialized view now properly finds associated
+  indexes for automatic reapplication. An issue in the previous index query was
+  returning no indexes.
+- The generators now accept namespaced view definitions. For example: `rails
+  generate scenic:view my_app.users`.
```

Comment:
I don't really care either. Bigger numbers are nicer and it feels like a feature but :+1: either way.